src/V3Const.cpp

```diff
@@ -945,6 +945,21 @@ class ConstVisitor final : public VNVisitor {
 
     // METHODS
 
+    void deleteVarScopesUnder(AstNode* subtreep) {
+        if (!subtreep) return;
+        AstScope* const scopep = VN_CAST(const_cast<AstNode*>(m_scopep), Scope);
```

Comment:
Seems unnecessary, just use m_scopep directly?

Reply:
The function iterates over variables in scopep below. m_scopep is a AstNode